src/main/java/jalau/at18/kingoftokyo/Card.java

```diff
@@ -0,0 +1,18 @@
+package jalau.at18.kingoftokyo;
+
+public abstract class Card {
+    private String name;
```

Comment:
BTW, you will probably get a checkstyle error if you use protected, we can update checkstyle rules to allow this. Please modify `checkstyle.xml` in line 174 to include the following:
```
<module name="VisibilityModifier">
      <property name="protectedAllowed" value="true" />
</module>
```
This will let you to have protected attribues

Reply:
We changed the private for protected attributes, and modified the checkstyle config